<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>705</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Blockbusted</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>1366</width>
     <height>647</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>10000</width>
     <height>10000</height>
    </size>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>20</y>
      <width>451</width>
      <height>391</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="title">
     <string>Admin Rights</string>
    </property>
    <widget class="QSplitter" name="splitter_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>96</width>
       <height>92</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QPushButton" name="add_emp">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>150</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>300</width>
        <height>150</height>
       </size>
      </property>
      <property name="text">
       <string>Add Employee</string>
      </property>
     </widget>
     <widget class="QPushButton" name="remove_emp">
      <property name="text">
       <string>Remove Employee</string>
      </property>
     </widget>
     <widget class="QPushButton" name="view_emp">
      <property name="text">
       <string>View Employees</string>
      </property>
     </widget>
     <widget class="QPushButton" name="edit_Inv">
      <property name="text">
       <string>Edit Inventory</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>341</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Left Menu</string>
    </property>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>120</width>
       <height>154</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QPushButton" name="check_in">
      <property name="text">
       <string>Check-In</string>
      </property>
     </widget>
     <widget class="QPushButton" name="check_out">
      <property name="text">
       <string>Check-out</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_Customer">
      <property name="text">
       <string>Add Customer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="remove_Customer">
      <property name="text">
       <string>Remove Customer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="view_Customers">
      <property name="text">
       <string>View Customers</string>
      </property>
     </widget>
     <widget class="QPushButton" name="view_Inv">
      <property name="text">
       <string>View Inventory</string>
      </property>
     </widget>
     <widget class="QPushButton" name="quit_Button">
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
